packages/proxy/src/proxy.ts

```diff
@@ -305,59 +336,72 @@ export async function proxyV1({
   const isStreaming = !!bodyData?.stream;
 
   let stream: ReadableStream<Uint8Array> | null = null;
-  if (useCache) {
+  if (readFromCache) {
     const cached = await cacheGet(encryptionKey, cacheKey);
 
     if (cached !== null) {
-      cacheHits.add(1);
       const cachedData: CachedData = JSON.parse(cached);
+      // XXX simplify once all cached data has a timestamp - assume existing data has age of 7 days
+      const ttl = cachedData.metadata?.ttl ?? DEFAULT_CACHE_TTL;
```

Comment:
shouldn't we compute this as something like `expirationTime - now`, unless I'm mistaken?

Reply:
this variable name is misleading - the response directive for max-age is the amount of time the response is fresh from the moment if was generated. ill rename

Reply:
ahhh i see, i was mistaken. this makes sense. we could probably also then change the parameterization back to cached_at and total ttl? sorry!